qiskit/circuit/gate.py

```diff
@@ -120,7 +121,7 @@ def control(
         return add_control(self, num_ctrl_qubits, label, ctrl_state)
 
     @staticmethod
-    def _broadcast_single_argument(qarg: List) -> List:
+    def _broadcast_single_argument(qarg: list[Any]) -> Iterator[tuple[list[Any], list[Any]]]:
```

Comment:
Here you use `list[Any]` but elsewhere you use `list`. Is there a difference? If not then let's just do `list` everywhere.